### PR TITLE
document migrating spatial types with DMS

### PR DESCRIPTION
**Pull request by taroface**
[DOC-6040](https://cockroachlabs.atlassian.net/browse/DOC-6040)
DOC-9243

@Jeremyyang920 I took a best guess at what the ticket (above link) was describing.

### CODE REVIEW

**Review by Jeremyyang920 — COMMENTED** [review]
I have no idea about how accurate this is since im not really in the loop on any of the spacial stuff. Perhaps adam might though.

File: src/current/v23.2/aws-dms.md [hunk 1]
```diff
@@ -45,7 +45,9 @@ Complete the following items before starting the DMS migration:
...
+
+        - [Spatial data types]({% link {{ page.version.version }}/spatial-data-overview.md %}) can have different input and output formats, which may create problems for the DMS migration. To import spatial data columns, you can create a [computed column]({% link {{ page.version.version }}/computed-columns.md %}) on the source database that uses the [`st_asbinary()` function]({% link {{ page.version.version }}/functions-and-operators.md %}#spatial-functions) to convert the spatial type into [Well Known Binary (WKB)]({% link {{ page.version.version }}/well-known-binary.md %}) format. Then use a [transformation rule](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Tasks.CustomizingTasks.TableMapping.SelectionTransformation.Transformations.html) to migrate the computed column to a CockroachDB computed column that uses another [spatial function]({% link {{ page.version.version }}/functions-and-operators.md %}#spatial-functions), such as `st_geomfromwkb()`, to convert the WKB values back to the appropriate spatial type.
```
Comment (rharding6373) [inline src/current/v23.2/aws-dms.md:50]: Why do we recommend converting to WKB and not EWKB? WKB only support 2D geo types. EWKB can support more dimensions: https://libgeos.org/specifications/wkb/

To do this you'd use `st_asewkb()` instead of `st_asbinary` and `st_geomfromewkb()` instead of `st_geomfromwkb()` in your example.

This may be more of a question for the migrations team. I haven't tested this.

Comment (taroface) [reply src/current/v23.2/aws-dms.md:50]: I'm not sure if there was a specific reason for recommending WKB over EWKB. @Jeremyyang920 do you think we can find a time to test the workaround?

Comment (Jeremyyang920) [reply src/current/v23.2/aws-dms.md:50]: Can see when I have time to test this out, but its probably low in priority on list of things to do. But also wonder how many customers have ran into such an issue before that its worth adding to the docs right now.

Comment (taroface) [reply src/current/v23.2/aws-dms.md]: Understood. In the interest of getting this PR off all our plates, I've amended the guidance to use EWKB (@rharding6373, PTAL) and commented out the guidance so that it won't appear on the site. When we have time to test this workaround, I can open another PR to unhide the guidance.

**Review by rharding6373 — APPROVED** [review]

File: src/current/v23.2/aws-dms.md (hunk 1 above)
Comment (rharding6373) [reply src/current/v23.2/aws-dms.md]: SGTM, thanks for updating.

File: src/current/v22.2/aws-dms.md [hunk 2]
```diff
@@ -45,7 +45,11 @@ Complete the following items before starting the DMS migration:
...
 
-    - Ensure that any schema changes are also reflected on your tables, or add [transformation rules](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Tasks.CustomizingTasks.TableMapping.SelectionTransformation.Transformations.html). If you make substantial schema changes, the AWS DMS migration may fail.
+    - Ensure that any schema changes are also reflected on your target tables, or add [transformation rules](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Tasks.CustomizingTasks.TableMapping.SelectionTransformation.Transformations.html) to your [table mappings](#step-2-3-table-mappings). If you make substantial schema changes, the AWS DMS migration can fail.
+
+        {% comment %}
```
Comment (Amruta-Ranade) [inline src/current/v22.2/aws-dms.md:50]: Should this be uncommented? Same question for the other docs

Comment (taroface) [reply src/current/v22.2/aws-dms.md:50]: Sorry for the confusion @Amruta-Ranade, the comment explaining this was resolved, but here's the thread: https://github.com/cockroachdb/docs/pull/18325#discussion_r1508024865

TLDR this is intentional and the guidance can be uncommented once it is re-tested by the team!

Comment (Amruta-Ranade) [reply src/current/v22.2/aws-dms.md:50]: Ah, gotcha. Thanks for the clarification!

**Review by Amruta-Ranade — APPROVED** [review]
